js/script.js

```diff
@@ -220,20 +252,34 @@ function popwork(id) {
     <li class="hcj">
       <p>${cards[id].technologies.js}</p>
     </li>
+    <li class="hcj">
+      <p class ="hide">${cards[id].technologies.Github}</p>
+    </li>
+    <li class="hcj">
+      <p class ="hide">${cards[id].technologies.Ruby}</p>
+    </li>
+    <li class="hcj">
+      <p class ="hide">${cards[id].technologies.Bootstrap}</p>
+    </li>
   </ul>
+  <div class= "hr">  </div>
   <div class="see-pop">
-  <button class="button" type="button"}>
+  <button class="press" type="button"}>
   <a href='${cards[id].buttons.seelive}'> See Live </a>
   <img src="./Icons/seeLive.png">
   </button>
-  <button class="button" type="button"}>
+  <button class="press" type="button"}>
   <a href='${cards[id].buttons.seesource}'>See source</a>
   <img src="./Icons/seeSource.png">
   </button>
+  </div>
+  </div>
   </div >
   </div>
   `;
   mobilePopup.appendChild();
   mobilePopup.classList.toggle('hidden');
+  // eslint-disable-next-line no-undef
```

Comment:
- [ ] Disabling linters is not really good because it doesn't allow an ethical coding practice that conforms you with industry standards. Kindy remove these lines and see how you can resolve the issues by reading what the message is :wink: